app/javascript/components/patient/Patient.js

```diff
@@ -13,9 +13,18 @@ import FollowUpFlagPanel from './follow_up_flag/FollowUpFlagPanel';
 import FollowUpFlagModal from './follow_up_flag/FollowUpFlagModal';
 import { navQueryParam, patientHref } from '../../utils/Navigation';
 
+let rootHeaderLevel;
+
+// The Heading element below throws a false-positive for prop-types validation
+/* eslint-disable-next-line react/prop-types */
+const Heading = ({ level, children, ...props }) => {
+  return React.createElement(`h${Math.min(level, 6)}`, props, children);
+};
```

Comment:
I'd prefer this go in a utils file, as we will certainly use it in other components in the future.

Reply:
added.